go.mod

```diff
@@ -1,12 +1,6 @@
 module github.com/SAP/jenkins-library
 
-go 1.21
-
-toolchain go1.21.9
-
-// It is a locked dependency of github.com/buildpacks/lifecycle@v0.18.4. The maintainers may remove the lock
-// in future releases. Check if 'replace' statement still there in their go.mod file. Remove line below if not.
-replace github.com/moby/buildkit => github.com/moby/buildkit v0.11.6
+go 1.22.4
```

Comment:
We need to update version in github workflows as well. You can find them be keyword `1.21`

Reply:
and here also: https://github.com/SAP/jenkins-library/blob/master/src/com/sap/piper/PiperGoUtils.groovy#L33